registration_ref/app.py

```diff
@@ -46,6 +46,9 @@ def create_in_foundries(client_cert: str, api_token: str, name: Optional[str] =
         "OSF-TOKEN": api_token,
     }
     for x in (0.1, 0.2, 1, 0):
+        log.info("Initiating request")
+        log.info("Using DEVICE_REGISTRATION_API: %s", Settings.DEVICE_REGISTRATION_API)
+        log.info("Using VERIFY_SSL: %s", Settings.VERIFY_SSL)
         r = requests.put(
             "https://api.foundries.io/ota/devices/", headers=headers, json=data
```

Comment:
Don't you want to use these new settings here?

After that, I don't think we need those logs.

Reply:
The logs are helpful for knowing the values used and know what is wrong when it fails.
But if you prefer that we remove it then that is fine.

Reply:
But if you absolutely hate it, then I can remove it.

Reply:
It would be nice to have these settings logged once at the app startup, as they are global.

Reply:
OHHH that is a mistake.
I did the changes in another branch I missed copy the line.
Good catcher.